src/WingetCreateCore/Common/GitHub.cs

```diff
@@ -479,5 +496,134 @@ private async Task DeletePullRequestBranch(int pullRequestId)
                 await this.github.Git.Reference.Delete(this.wingetRepoOwner, this.wingetRepo, newBranchNameHeads);
             }
         }
+
+        private static class GitHubManifestMetadata
+        {
+            public static async Task PopulateManifestMetadata(Manifests manifests, string serializerFormat, GitHubClient client)
+            {
+                // Get owner and repo from the installer manifest
+                GitHubUrlMetadata? metadata = GetMetadataFromGitHubUrl(manifests.InstallerManifest);
+
+                if (metadata == null)
+                {
+                    // Could not populate GitHub metadata.
+                    return;
+                }
+
+                string owner = metadata.Value.Owner;
+                string repo = metadata.Value.Repo;
+                string tag = metadata.Value.ReleaseTag;
+
+                var githubRepo = await client.Repository.Get(owner, repo);
+                var githubRelease = await client.Repository.Release.Get(owner, repo, tag);
+
+                // License
+                if (string.IsNullOrEmpty(manifests.DefaultLocaleManifest.License))
+                {
+                    // License will only ever be empty in new command flow
+                    manifests.DefaultLocaleManifest.License = githubRepo.License?.SpdxId ?? githubRepo.License?.Name;
+                }
+
+                // ShortDescription
+                if (string.IsNullOrEmpty(manifests.DefaultLocaleManifest.ShortDescription))
+                {
+                    // ShortDescription will only ever be empty in new command flow
+                    manifests.DefaultLocaleManifest.ShortDescription = githubRepo.Description;
+                }
+
+                // PackageUrl
+                if (string.IsNullOrEmpty(manifests.DefaultLocaleManifest.PackageUrl))
+                {
+                    manifests.DefaultLocaleManifest.PackageUrl = githubRepo.HtmlUrl;
+                }
+
+                // PublisherUrl
+                if (string.IsNullOrEmpty(manifests.DefaultLocaleManifest.PublisherUrl))
+                {
+                    manifests.DefaultLocaleManifest.PublisherUrl = githubRepo.Owner.HtmlUrl;
+                }
+
+                // PublisherSupportUrl
+                if (string.IsNullOrEmpty(manifests.DefaultLocaleManifest.PublisherSupportUrl) && githubRepo.HasIssues)
+                {
+                    manifests.DefaultLocaleManifest.PublisherSupportUrl = $"{githubRepo.HtmlUrl}/issues";
+                }
+
+                // Tags
+                manifests.DefaultLocaleManifest.Tags ??= githubRepo.Topics?.ToList();

```

Comment:
Some repositories contain more than 16 tags, yet the schema permits only 16. This can occasionally lead to issues with manifest validations. Therefore, it's necessary to exclude the additional tags.

Reply:
Fixed in https://github.com/microsoft/winget-create/pull/543/commits/23c189af7ec38f55d8efe9e9147d1c6a9a9cff9b